File: Proyects/UniJava/TP4/ex3/Circle.java
public class Circle extends Ellipse implements Movable{
	private Point center;
	private double radius;

	public Circle(Point center, double radius){
		super(center, radius, radius);
		this.center = center;
		this.radius = radius;
	}

	@Override
	public String toString(){
		String ret = "[Center{" + this.center + "},Radius{" + this.radius + "}]";
		return ret;
	}
}

File: Proyects/UniJava/TP9/ex4/circle.rb
require_relative 'ellipse'
class Circle < Ellipse
  def initialize(center, radius)
    super(center, radius * 2, radius * 2)
  end

  def ==(other)
    return false unless other.is_a? Circle
    @center == center && @diameter_x == other.diameter_x
  end

  def hash
    [@center, @diameter_x].hash
  end
end
